### PR TITLE
Initial revision of documentation for contributors

### PR DESCRIPTION
**Pull request by JeremyMcCormick**
This is the initial version of documentation for contributors.

### CODE REVIEW

**Review by molinaro-m — APPROVED** [review]
I skimmed through the text.
I think that it is fine, at least for now.
Maybe a little too much for an entry user, but on Wdnesday we will have feedback on that.

**Review by gmantele — COMMENTED** [review]
Maybe it would useful for the users to know a bit about the repository structure, and especially in which directory are the files to change or add (i.e. `content/`). I don't think we should invite them to change anything in `layout/` for instance.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# Contributing
+
+The new version of the IVOA website is being developed with [Hugo](https://gohugo.io/), a popular open-source tool for generating static websites. Instead of creating and modifying the site content directly in HTML, contributors will use markdown which is then converted. Since the site is now managed in Github using source control, development will follow a typical Git workflow, which will be covered in detail.
+
+# Development Workflow
+
+## Install Git
+
+If you do not have it already, then you will need to [install git](https://github.com/git-guides/install-git) in order to work on the website locally.
```
[the remark this answers is not in the document]
Comment (JeremyMcCormick) [reply CONTRIBUTING.md:9]: Editing files directly in Github seems okay for minor changes like fixing mispellings. I would not encourage it for anything major like adding new pages or major rewrites of pages.

Probably I wouldn't include it in the contributing doc as an option. I would still prefer the PR procedure, even for small changes.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+You may additionally include some brief text in the branch name to describe what you plan on doing, such as `iss1234-add-a-page-about-the-ivoa`. The most important part is starting the branch name with `iss` followed by the number of the issue in the tracker.
+
+# Working with Hugo
+
+## Installing Hugo
+
+Hugo is required for building the site locally to check and preview your changes. You may follow the [installation instructions](https://gohugo.io/installation/) for your platform.
+
+A quick procedure to download Hugo on Linux from the [release page](https://github.com/gohugoio/hugo/releases) is:
```
[the remark this answers is not in the document]
Comment (JeremyMcCormick) [reply CONTRIBUTING.md:61]: Yes, I think that could be useful.

**Comment by JeremyMcCormick** [comment]
> Maybe it would useful for the users to know a bit about the repository structure, and especially in which directory are the files to change or add (i.e. `content/`). I don't think we should invite them to change anything in `layout/` for instance.

Agreed - I'll try to add some additional text indicating where the content pages should go.

**Comment by JeremyMcCormick** [comment]
> I skimmed through the text.
> 
> I think that it is fine, at least for now.
> 
> Maybe a little too much for an entry user, but on Wdnesday we will have feedback on that.

I agree that it could be overwhelming for an entry level user. I am thinking maybe that the sections on Git and Github could be streamlined by pointing to some external documentation, e.g., how to make a PR.

It's also possible that the local `hugo` section can essentially be marked as optional. With the versioned sites working now, it would be possible to just check changes there rather than needing to do it locally.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gmantele) [reply CONTRIBUTING.md:9]: I agree, it is not ideal as not really an as complete solution as a clone of the repository. Although, creating a new page is not that complicating or risky, but ok, let's already start with the clone solution.

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gmantele) [reply CONTRIBUTING.md:61]: I'll try to prepare some PR then.

**Pull request merged**